[docs] Add Using Analytics guide (#19637)
expo/expo: docs/pages/guides/using-analytics.mdx

---
title: Using Analytics
---

import { BoxLink } from '~/ui/components/BoxLink';

> Analytics in your app allow you to measure how users behave when they use your app and make informed decisions. It is done using an Analytics provider that enables you to track common events and define your own custom events. After collecting data, you can analyze your users' interactions with the app. It can be a key to providing a better user experience.

Reviewer: ```suggestion
An analytics service allows you to track how users interact with your app. With this data, you can take a measured approach when improving your app.
```

The following list provides common analytics providers that are available in the Expo and React Native ecosystem.

> > Each analytics SDK requires configuring custom native code. Native code is not configurable when using Expo Go for development. However, you can create a [development build](/development/introduction/) for your Expo project that allows configuring native code.

Reviewer: ```suggestion
> Each analytics SDK requires configuring custom native code. Native code is not configurable when using Expo Go for development. However, you can create a [development build](/development/introduction/), which will allow you to use any of the services below.
```
